packages/appeals-service-api/src/repositories/document-metadata-repository.js

```diff
@@ -14,22 +14,41 @@ class DocumentMetadataRepository extends MongoRepository {
 	 * @param {DocumentType} documentType
 	 * @return {Promise<any>}
 	 */
-	async getDocumentMetadata(caseRef, documentType) {
-		try {
-			return await this.findOneByQuery(
-				{
-					caseRef: caseRef,
-					documentType: documentType,
-					virusCheckStatus: 'scanned',
-					redactedStatus: 'redacted',
-					publishedStatus: 'published'
-				},
-				{
-					version: -1
-				}
-			);
-		} catch (e) {
-			console.log(e);
+	async getDocumentMetadata(caseRef, documentType, returnMultipleDocuments) {
+		if (returnMultipleDocuments == 'true') {
```

Comment:
As mentioned it would be good if we guarantee this is boolean so we can do if(returnMultipleMultiple)